Adding scripts to create, insert,delete schema. Adding jupyter notebo…

ShiveeUSF opened the pull request:
…ok to connect to DB and execute scripts

ShiveeUSF committed:
Adding scripts to create, insert,delete schema. Adding jupyter notebook to connect to DB and execute scripts

Shulun commented on delete_script.sql:
```diff
@@ -0,0 +1,7 @@
+drop table appdata.vote;
```
Great work! Could we change all the key words to capital letters? This way it is more consistent and standardized.
For example `drop table` to `DROP TABLE`.

bowema replied:
Updated

Shulun commented on delete_script.sql:
```diff
@@ -0,0 +1,7 @@
+drop table appdata.vote;
+drop table appdata.poll;
+drop table appdata.user;
+DROP SEQUENCE poll_image_id_seq;
```
I don't think we really need to add any sequence column here, I would prefer that our python script takes care of generating the  unique sequence that serves as the id for each row. We could do it like this:
```
import uuid
print(uuid.uuid4())
```

ShiveeUSF replied:
We shall retain sequence just for image ids.

bowema replied:
removed, but should we keep it for image ids?

Shulun commented on schema.sql:
```diff
@@ -0,0 +1,52 @@
+ );
+
+
+CREATE SEQUENCE if not exists poll_image_id_seq;
```
Like I mentioned above, I still feel it is easier to control id sequence in python :). Nonetheless, great thinking and awesome work!

bowema replied:
lgtm

Shulun commented on schema.sql:
```diff
@@ -0,0 +1,52 @@
+
+CREATE TABLE IF NOT EXISTS appdata.vote (
+	user_id bigint REFERENCES appdata.user(user_id) ON DELETE CASCADE
+	,voter_poll_id bigint
```
Please fix the comma spacing so everything is consistent here across all scripts.

bowema replied:
fixed

Shulun commented on connect_to_db.ipynb:
```diff
@@ -0,0 +1,232 @@
+{
```
Could you add the python scripts taken out of the jupyter notebook for executing the sql scripts in the repo?

ShiveeUSF replied:
Since there is a lot of back and forth currently and we need to complete other parts of the project as well, I can fix these things later. For now lets just focus on getting main functionality done.

ShiveeUSF replied:
will do that later. We are still changing a lot of things

bowema replied:
lgtm

bowema replied:
Deleted

Shulun commented on schema.sql:
```diff
@@ -0,0 +1,52 @@
+  , is_developer boolean
+  , created_on timestamp without time zone NOT NULL
+  , last_login timestamp without time zone NOT NULL
+  , PRIMARY KEY (user_id)
```
I would suggest that we have a primary key of (uuid, email, username) instead. user_id -> uuid, name -> username

ShiveeUSF replied:
done currently primary key is composite (uuid,username)

ShiveeUSF committed:
Modified DB script. Fetching password from env. Creating user and poll table

ShiveeUSF committed:
Adding Shulun'code

Merge branch 'master' of https://github.com/productanalyticsteam9/back_end into shivee

ShiveeUSF committed:
Python scipt to connect to DB and create schema and user and poll table

ShiveeUSF committed:
Notebook that loads trained model, takes new image and prints tag

bowema committed:
delete files

bowema committed:
change to UPPER CASE and add quotes for table names

bowema committed:
delete sequence variable

bowema committed:
formatting - add quotes and capitalize;

bowema committed:
adjust spaces to be consistent

bowema commented:
Updated all Shulun's points.

Commit:
Merge branch 'master' of https://github.com/productanalyticsteam9/back_end into shivee

Make sure shivee branch is synced up with the master branch.

Commit:
Re-organized the sql codes into a sub-folder
